asynctools/asyncpipe.nim

```diff
@@ -32,6 +32,7 @@
 ##   close(o)
 
 import asyncdispatch, os
+import asyncpty
```

Comment:
This import should not be present, `asyncpipe` must not depend on `asyncpty`.

Reply:
```suggestion
```